plugins/00_dokku-standard/subcommands/run

```diff
@@ -23,6 +24,18 @@ dokku_run_cmd() {
   [[ "$DOKKU_RM_CONTAINER" ]] && local DOKKU_RUN_OPTS="--rm"
   has_tty && local DOKKU_RUN_OPTS+=" -i -t"
   is_image_herokuish_based "$IMAGE" && local EXEC_CMD="/exec"
+
+  DOKKU_QUIET_OUTPUT=1 extract_procfile "$APP"
+
+  POTENTIAL_PROCFILE_KEY="$1"
+  PROC_CMD=$(get_cmd_from_procfile "$APP" "$POTENTIAL_PROCFILE_KEY" || echo '')
+  remove_procfile "$APP"
```

Comment:
I'm now cleaning up the Procfile after using it, just like `deploy` does.

It would be nice if `get_cmd_from_procfile` would take care of extracting the Procfile and cleaning it up. Potential downside of that is that if you would call it multiple times it would extract the Procfile every time, but the overhead of that is pretty small I think and also seems unlikely you would call this multiple times. What do you think @josegonzalez @michaelshobbs?